README.md

```diff
@@ -1,13 +1,37 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
+[![examples](https://img.shields.io/badge/learn-examples-blue)](https://learn.fragcolor.xyz/)
 
-## Tutorials
+This repository is a collection of example games and tutorials using the [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) programming language.
+
+Chainblocks is inspired by lisp and hence is an extremely powerful as well as fun to learn language.
+
+These tutorials are designed to help you learn practical Chainblocks programming step-by-step. Each tutorial covers a few langauge features, blocks, and functions in the context of building a game from scratch.
+
+## Installation
+
+[Set up your environment](https://docs.fragcolor.xyz/contribute/code/getting-started/) and [build Chainblocks](https://docs.fragcolor.xyz/contribute/code/building-chainblocks/) before working through the tutorials given in this repository.
+
+## Usage
+
+Though there are no hard pre-requisites for these tutorials, if you're new to programming you should read the [Chainblocks primer](https://learn.fragcolor.xyz/how-to/chainblocks-primer/) first.
+
+It's also recommended to go through the [Chainblocks architecture](https://docs.fragcolor.xyz/architecture-guides/chainblocks) guide to understand what's going on under the hood.
+
+### Tutorials
 
 * [Snake](./tutorials/snake/)
```

Comment:
I think we should link to the website page instead. here it opens the folder in GitHub but the markdown files are not meant to be read from there if someone wants to follow the tutorial.

```suggestion
* [Snake](https://learn.fragcolor.xyz/snake/)
```

Reply:
Ah! you're right - didn't notice that. Fixed now.